great-idea/css/index.css

```diff
@@ -63,4 +63,52 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+body {
+	margin: 0px 200px 0px 200px;
+}
```

Comment:
it's better to use a container to set these things and just let the body be body. also, great use of TRBL! but if the values are the same, you can just do '0 200px' instead of all four of them